2020.4 blog post (#670)
tableau/connector-plugin-sdk: _posts/2020-12-14-new-release-2020-4.md

---
title:  "Connector SDK now targets Tableau 2020.4"
abstract: "The master SDK branch now targets Tableau 2020.4. The Connector SDK for 2020.3 has released for those wishing to target that version."
---

On December 14, with the full release of Tableau 2020.4 we moved the latest version of the master branch to target Tableau 2020.4. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2020.3, please download the [Tableau Connector SDK for Tableau 2020.3](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2020.3) release.

> Features included now included in the SDK for Tableau 2020.4 include:

Reviewer: Nit: Too many includes :) 

```suggestion
Features added to the SDK for Tableau 2020.4 include:
```

- __ISO8601 support added__ <br/>
 ISO8601 support has been added to the postgres_odbc and postgres_jdbc samples. The samples can be found [here](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins).
- __JDBC Kerberos Postgres sample using Connection Dialogs v2__ <br/>
This sample connector plugin demonstrates how to incorporate different Kerberos-based authentication modes for the "Tableau + Postgres + JDBC" use cases. This plugin has been created using the [Connection Dialog v2 framework]({{ site.baseurl }}/docs/mcd) See more [here](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/scenarios/jdbc_kerberos).
- __Partner Run Connectors Tests__ <br/>

> Added tests that the pastner can run to validate certian aspects of the connector developed. More information about running the tests can be found in the "Run Connectors tests" section [here](https://tableau.github.io/connector-plugin-sdk/docs/tdvt).

Reviewer: Spelling error: pastner->partner
